### PR TITLE
Redo PC Live Operating Systems section

### PR DESCRIPTION
**Pull request by dawidpotocki**
Remove the following operating systems, because of described issues

KNOPPIX:
  - Site has optional HTTPS, so all links, will make you go back to HTTP
  - Source code distributed on ISOs, no online repositories/tarballs

Puppy Linux:
  - Site is only over HTTP
  - Looks like garbage

Tiny Core Linux:
  - Site is only over HTTP
  - Not really usable or useful
  - Looks like garbage

Add Debian and Fedora to Worth Mentioning PC Live Operating Systems,
since they are recommended as normal operating systems and support
LiveUSB.

---

https://deploy-preview-1600--privacytools-io.netlify.com/operating-systems/#live_os

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
           </summary>
           <span class="nav-dropdown">
             <a class="dropdown-item" href="/operating-systems/#os"><span class="fas fa-th-large fa-fw"></span> PC OS</a>
-            <a class="dropdown-item" href="/operating-systems/#live_os"><span class="fas fa-compact-disc fa-fw"></span> Live CD OS</a>
+            <a class="dropdown-item" href="/operating-systems/#live_os"><span class="fas fa-compact-disc fa-fw"></span> PC Live OS</a>
```
Comment (Mikaela) [inline _includes/nav.html]: I am not sure if the PC is a necessary word.

File: _includes/sections/live-operating-systems.html [hunk 2]
```diff
@@ -10,30 +10,17 @@ <h1 id="live_os" class="anchor"><a href="#live_os"><i class="fas fa-link anchor-
...
-
 <h3>Worth Mentioning</h3>
 
 <ul>
-  <li><a href="https://distro.ibiblio.org/tinycorelinux/">Tiny Core Linux</a> <span class="badge badge-info">GNU/Linux</span> <span class="badge badge-warning" data-toggle="tooltip" title="This software may depend on or recommend non-free software.">contrib <i class="far fa-question-circle"></i></span> - A minimal Linux operating system focusing on providing a base system using BusyBox and FLTK. The distribution is notable for its size (15 MB) and minimalism, with additional functionality
-    provided by extensions.</li>
+  <li>
+    <a href="https://getfedora.org/">Fedora Workstation</a>
+    <span class="badge badge-info">GNU/Linux</span>
+    - Fedora is a Linux distribution developed by the Fedora Project and sponsored by Red Hat. Fedora Workstation is a secure, reliable, and user-friendly edition developed for desktops and laptops, using GNOME as the default desktop environment.
```
Comment (Mikaela) [inline _includes/sections/live-operating-systems.html]: Should the spins be mentioned?

File: _includes/nav.html (hunk 1 above)
Comment (dawidpotocki) [reply _includes/nav.html]: Well, you could make a mobile live operating system.

File: _includes/sections/live-operating-systems.html (hunk 2 above)
Comment (dawidpotocki) [reply _includes/sections/live-operating-systems.html]: Lol nope.

**Pull request merged**